src/ray/util/tests/pipe_logger_test.cc

```diff
@@ -375,5 +311,3 @@ TEST(PipeLoggerCompatTest, CompatibilityTest) {
 }  // namespace
 
 }  // namespace ray
```

Comment:
Can you enable `stream_redirection_utils_test` for windows?

Reply:
yes